src/modules/Progress/Progress.js

```diff
@@ -140,10 +140,9 @@ class Progress extends Component {
   }
 
   showProgress = () => {
-    const { label, precision, progress, total, value } = this.props
+    const { label, precision, progress } = this.props
 
     if (label || progress || !_.isUndefined(precision)) return true
```

Comment:
Sure. I noticed it, but I didn't want to make changes that were not directly related to the task. I'll do that now